app/assets/javascripts/select-kit/addon/components/selected-choice.js

```diff
@@ -23,4 +23,8 @@ export default Component.extend(UtilsMixin, {
   itemName: computed("item", function () {
     return this.getName(this.item);
   }),
+
+  readOnly: computed("item", function () {
+    return this.get("mandatoryValues")?.split("|")?.includes(this.item.id);
```

Comment:
Probably put `this.get("mandatoryValues")?.split("|")?` into another computed property like `mandatoryValuesArray:` then here you can do `this.mandatoryValuesArray.includes(this.item.id)` and you're not splitting the string every time.